src/pages/guides/review/index.md

```diff
@@ -35,28 +57,84 @@ By creating a project using the SDK in Adobe Developer Console, you are agreeing
+
+If your app processes personal information, include a link to the privacy policy on your app’s website, per the Adobe Developer Agreement.
+
+### 4. Details for Approval
+
+Describe what your app does, to help Adobe review it for approval. Allowing other enterprises to discover your app or integration in the Adobe Admin Console will make it so that the administrators of those enterprises can manage access to it for their users. 
 
-1. Navigate to your project in Adobe Developer Console -- select "Submit for review" from the __Project overview__ tab or select the __Approval__ tab on the left panel.
-   ![Submit for review in Console UI](submission.png)
-2. If this is your first Adobe integration, set up your public profile - note that the information that appears on your public profile will appear in the consent screen displayed to users.
+### 4. Notes to reviewer
 
-3. Fill out the fields in the __Consent Screen Details__ section.
+<InlineAlert variant="error" slots="header, text" />
 
-4. Fill out the fields in the __Approval Details__ section. In the __Notes to reviewer__ field, make sure to:
-   * Provide steps for our review team to test the integration on your platform. If free sign-up is not available, please provide a set of working test credentials for our review team.
-   * Please note __what version__ of the SDK you are using. At this time we are only accepting v3 and above.
-   * Please note whether your app uses the full editor, quick actions, and/or modules.
-   * Please list which quick actions(s), if any, you use in your application.
-   * Please list which modules, if any, you use in your application.
-   * Please list the intended behavior of any custom export buttons you use with the full editor, modules, or quick actions.
-   * Confirm that the `appName` used to initialize the SDK in your application code reflects your app's name. Note any projects created through your integration will be saved in the user's Express account, inside an Express folder with this name.
-   * Confirm that the SDK is never initialized more than once per page load.
+IMPORTANT: Deprecation Warning
 
-5. Complete the __Screenshots for approval__ section.
-6. Once you think you're good to go, click the "Submit for Review" button. Our review team will ensure your integration fulfills all the guidelines and passes all the [test cases](#test-cases) before approving it for public usage.
+We are no longer approving integrations using v1 or v2 of the SDK - both versions will be deprecated in 2024.
 
-### Test cases
+Make sure you’ve included the following in the **Notes to reviewer** feild.
+
+1. **Test credentials** for your integration. 
+2. **Workflow** of your integration within your product (URL, step-by-step instructions, and more.)
+3. **Version** of the SDK you are using.
+4. List the intended behavior of any custom export buttons you use with the full editor, modules, or quick actions.
```

Comment:
Maybe add a menacing statement after the list, like: Failing to provide these details will sadly result in an automatic rejection, and you'll be shamed by your friends and family forever. Another alert box could be useful to draw attention.